### ISSUE TITLE
Copy Key to Clipboard on Click

### ISSUE THREAD

**Issue by markdenihan** [OWASP/SecurityShepherd#217]
Where possible

**Comment by markdenihan** [comment]
@ismisepaul is working on this and is nearly complete

**Comment by markdenihan** [comment]
I've noticed using this that it takes a little time to initialise. Sometimes when I click it right after the icon appears nothing is actually on my clipboard. @ismisepaul - Do you think it would initialize faster if we included the js/css includes in the <head> of levels rather than the write once method we're using (That I opted for originally *hands up*) in the function where the key is returned?

**Comment by markdenihan** [comment]
In the Daggercon, I ran a Security Shepherd Workshop with the latest build. This functionality was not working in Iceweasel and a few other browsers. The only stuff to be copied would be what was being displayed in the textarea. So if there was overspill, it would not be added to the clipboard.

Can you look into that @ismisepaul before the code freeze?

**Comment by markdenihan** [comment]
@ismisepaul - Fyi, I've moved the css ref element to inside the lessons and challenges instead of hash.java in commit 12f82806ad4e292639da9019e306048d22d96e43

**Comment by markdenihan** [comment]
@ismisepaul - Saw the new implementation you just put in, looks awesome. I removed the css ref elements from the lessons as they're defunct now (Pointing at octicons that no longer exists)

**Comment by markdenihan** [comment]
I've Fixed the CSS issues Paul's Implementation was having with the Tooltip that gives user feedback after they have clicked the "copy" button. I've moved the JS to be called by the level at start up rather than when we get the key, because it was taking 3 seconds extra to return the key on my local host instance. Now it's much snappier! Lastly I've ensured the Solution Key Click To Copy stuff is not encoded before sent to the browser, as there were some levels just returning a heap of Encoded HTML. This is now complete, @ismisepaul did all of the leg work. I'm just closing because he's in Iceland now :+1: 

 This will be a focus for Pre-Release Testing
